services-js/commissions-app/src/client/common/Checkbox.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+export interface Props {
+  name: string;
+  title: string;
+  value: string;
+  onChange: any;
+  onBlur: any;
+}
+
+export default class CheckboxElement extends React.Component<Props> {
+  render() {
+    return (
+      <div className="txt m-b300">
```

Comment:
The `className` of this `<div>` and the `<label>` below don't match patterns.boston.gov.